modules/extensions-support/src/test/java/org/mule/module/extension/internal/PasswordModelPropertyTestCase.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal;
+
+import static java.util.Arrays.asList;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import org.mule.api.registry.ServiceRegistry;
+import org.mule.extension.api.introspection.ExtensionFactory;
+import org.mule.extension.api.introspection.ExtensionModel;
+import org.mule.extension.api.introspection.ParameterModel;
+import org.mule.extension.api.introspection.declaration.fluent.Descriptor;
+import org.mule.extension.api.introspection.declaration.spi.ModelEnricher;
+import org.mule.extension.api.introspection.property.PasswordModelProperty;
+import org.mule.module.extension.internal.introspection.AnnotationsBasedDescriber;
+import org.mule.module.extension.internal.introspection.DefaultExtensionFactory;
+import org.mule.module.extension.internal.introspection.enricher.PasswordModelEnricher;
+import org.mule.module.extension.internal.runtime.connector.secure.SecureConnector;
+import org.mule.registry.SpiServiceRegistry;
+import org.mule.tck.junit4.AbstractMuleTestCase;
+import org.mule.tck.size.SmallTest;
+
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@SmallTest
+@RunWith(MockitoJUnitRunner.class)
+public class PasswordModelPropertyTestCase extends AbstractMuleTestCase
+{
+
+    private ExtensionFactory extensionFactory;
+    private ExtensionModel extensionModel;
+
+    @Before
+    public void setUp()
+    {
+        ClassLoader classLoader = getClass().getClassLoader();
+        ServiceRegistry serviceRegistry = mock(ServiceRegistry.class);
+        when(serviceRegistry.lookupProviders(ModelEnricher.class, classLoader)).thenReturn(asList(new PasswordModelEnricher()));
```

Comment:
better use the any(ClassLoader) matcher rather than depending on the test classloader. THis way this test won't become problematic if class loading handling changes